content/get-started/why-webpack.md

```diff
@@ -1,6 +1,31 @@
 ---
-title: Compared to Other Systems
+title: Why webpack?
+contributors:
+  - pksjce
+  - bebraw
+sort: 2
 ---
+
+You might be familiar with tools like Make, Grunt, Gulp, Browserify, or Brunch. Make is the oldest and the most known one as it's a general build tool and its history goes back to the 70s. More specialized *task runners*, like Grunt and Gulp, are focused on JavaScript and have healthy plugin ecosystems around them.
```

Comment:
When I read this page for the first time, I don’t know why “task runners” and “bundlers” have to be introduced here. My guess is that they are usually compared (webpack vs Gulp).

Would it make sense state that we’re going to draw a distinction? e.g.

> webpack is usually compared to tools like Make, Grunt, Gulp, Browserify or Brunch. However, some of these tools (Make, Grunt, and Gulp, which are *task runners*) have a much different purpose than webpack, which is a *module bundler*. Let’s draw a distinction between these categories of tools.